controllers/rcj_soccer_referee_supervisor/recorder/recorder.py

```diff
@@ -0,0 +1,41 @@
+from controller import Supervisor
+
+
+class VideoRecordAssistant:
+
+    def __init__(self, supervisor, resolution="720p"):
+        self.supervisor = supervisor
+        self.resolution = resolution
+
+        if not isinstance(self.supervisor, Supervisor):
+            raise TypeError()
+
+    def create_title(self):
```

Comment:
@aquahika The actual path should probably be specified as a parameter to the `Referee` or we can compute it from the variables that `Superviser` will probably know (like the names of the teams and so on).